lib/greed/score.rb

```diff
@@ -0,0 +1,26 @@
+module Greed
+  module Score
+    def calcScore(dice)
+      raise ArgumentError.new unless dice
+      score = 0
+
+      # Create hash of count of each die
+      counts = dice.inject(Hash.new(0)) {|hash, num| hash[num] += 1; hash}
+
+      # Calculate score
+      counts.each do |die, count|
+        trips, remainder = count / 3, count % 3
```

Comment:
Generally, you want one statement per line. Destructured assignment is great when you are presented with a complex return value, but in this case we're just doing two different things on the same line for the sake of brevity.

http://www.oualline.com/books.free/style/c04.html